tests/unit/helpers/ember-prop-types-test.js

```diff
@@ -0,0 +1,285 @@
+/**
+ * Unit test for the frost-power-display component
+ */
+
+// Because we want to satisfy ESLint rules for not using references that were not previously defined...
+// ...AND... we need to be able to spy on the same methods we need to import the "mocha" library twice,
+// which is why the second import of it disables the ESLint warning about such duplication on a per-line basis
+import {expect} from 'chai'
+import Ember from 'ember'
+import {afterEach, beforeEach, describe, it} from 'mocha'
+import * as spyOnMocha from 'mocha' // eslint-disable-line no-duplicate-imports
+import sinon from 'sinon'
+import validatePropTypes, * as PropTypesHelper from 'dummy/tests/helpers/ember-prop-types'
```

Comment:
In order to be able to spy on a method you must be able to access it off of an object, such as

`let spyExpect = sandbox.spy(PropTypesHelper.forTestingOnly, 'expect')`

without importing as currently are this spying is not possible